src/olympia/blocklist/utils.py

```diff
@@ -193,6 +193,34 @@ def split_regex_to_list(guid_re):
     return GUID_SPLIT.split(trimmed)
 
 
+def disable_addon_for_block(block):
+    """Disable appropriate addon versions that are affected by the Block, and
+    the addon too if 0 - *."""
+    from .models import Block
+    from olympia.reviewers.utils import ReviewFiles
+
+    review = ReviewFiles(
```

Comment:
At the moment it doesn't hurt to use `ReviewFiles`, but this is kinda dangerous. The rest of addons-server uses the `ReviewHelper` wrapper to get the right class, passing `version` parameter (and not just `None`).

Reply:
I looked at ReviewHelper but all it'd be used for is `set_review_handler` so it didn't seem much point.   The difficulty with passing in a `version` is knowing which version to provide - and `reject_multiple_versions` doesn't use `self.version` anyway for non-`human_review`.
Could if you want though.

Reply:
My worry is that if for instance we start overriding `reject_multiple_versions` in `ReviewUnlisted` for some reason, or maybe start having a `channel` property in the review classes to implement channel-specific behavior, this wouldn't get it. So I think this is worth fixing, but I agree it might be difficult. This could be a follow-up.

Reply:
https://github.com/mozilla/addons-server/pull/15435/commits/8136660bc6349399f90d97c04ccab358125c8c76